src/react-native-walkthrough-tooltip.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module 'react-native-walkthrough-tooltip' {
+  import React from 'react';
+  import { GestureResponderEvent, ViewStyle } from 'react-native';
+
+  export interface TooltipProps {
+    isVisible?: boolean;
+    content?: React.ReactElement;
+    placement?: 'top' | 'bottom' | 'left' | 'right' | 'center';
+    onClose?: (event: GestureResponderEvent) => void;
+    closeOnContentInteraction?: boolean;
+    closeOnBackgroundInteraction?: boolean;
+    backgroundStyle?: ViewStyle;
+    contentStyle?: ViewStyle;
+    tooltipStyle?: ViewStyle;
+    arrowStyle?: ViewStyle;
+    children?: React.ReactNode;
```

Comment:
Everything optional?

Reply:
Yes. If you check react-native-walkthrough-tooltip module, you'll find it.